aspnetcore/performance/performance-best-practices.md

```diff
@@ -42,7 +42,22 @@ A common performance problem in ASP.NET Core apps is blocking calls that could b
 
 A profiler, such as [PerfView](https://github.com/Microsoft/perfview), can be used to find threads frequently added to the [Thread Pool](/windows/desktop/procthread/thread-pools). The `Microsoft-Windows-DotNETRuntime/ThreadPoolWorkerThread/Start` event indicates a thread added to the thread pool. <!--  For more information, see [async guidance docs](TBD-Link_To_Davifowl_Doc)  -->
 
-## Return IEnumerable\<T> or IAsyncEnumerable\<T>
+## Return large collections
```

Comment:
Would it maybe be clearer to make the header what we want users to do instead of the dangerous scenario? Perhaps `Return large collections in multiple smaller pages` or something like that?